docs/tutorial/Split_Learning_for_bank_marketing.ipynb

```diff
@@ -95,18 +95,13 @@
   {
    "cell_type": "code",
    "execution_count": 1,
```

Comment:
this tutorial seems to be overwritten

Reply:
@zhangxingmeng refactored this tutorial.